android/src/com/mapswithme/maps/widget/menu/MyPositionButton.java

```diff
@@ -47,9 +48,17 @@ public void update(int mode)
   {
     mMode = mode;
     Drawable image = mIcons.get(mode);
-    @AttrRes int colorAttr =
-        mode == LocationState.FOLLOW || mode == LocationState.FOLLOW_AND_ROTATE || mode == LocationState.PENDING_POSITION
-        ? R.attr.colorAccent : R.attr.iconTint;
+    @AttrRes int colorAttr = R.attr.iconTint;
+    @DimenRes int sizeDimen = R.dimen.map_button_icon_size;
+    if (mode == LocationState.FOLLOW || mode == LocationState.FOLLOW_AND_ROTATE || mode == LocationState.PENDING_POSITION)
+    {
+      colorAttr = R.attr.colorAccent;
+      if (mode == LocationState.PENDING_POSITION)
+        sizeDimen = R.dimen.map_button_size;
+      else
+      sizeDimen = R.dimen.map_button_arrow_icon_size;
+
```

Comment:
indent and remove empty string

Reply:
Fixed the indent but what do you mean by empty string?

Reply:
Empty line?